contentcuration/contentcuration/frontend/shared/views/TipTapEditor/TipTapEditor/TipTapEditorStrings.js

```diff
@@ -224,7 +224,7 @@ const MESSAGES = {
     context: 'Accessibility label for the button that opens the file picker.',
   },
   supportedFileTypes: {
-    message: 'Supported file types: png, jpg, jpeg, svg, webp',
+    message: 'Supported file types: ',
```

Comment:
I think we should probably use a place holder here, in case in translation appending the list to the string is not what is required.

```suggestion
    message: 'Supported file types: { extensions }',
```